src/mumble/Log.cpp

```diff
@@ -333,35 +333,38 @@ void Log::clearIgnore() {
 	qmIgnore.clear();
 }
 
+/// Estimate the size of buf when encoded as an HTML image tag
+/// with the src attribute set to a base64-encoded data URL, such
+/// as:
+///
+///    <img src="image/jpeg;base64,[base64-of-buf-goes-here]" />
+///
+static unsigned int estimatedDataURLImageSize(const QByteArray &buf) {
+    // We estimate the extra characters (HTML) in the encoded
+    // message to be around 64.
+    const unsigned int htmlExtras = 64U;
```

Comment:
Isn't the only variable in this calculation the size of the mimetype? It seems to me the only possible difference in size would be "image/jpeg" vs. "image/png"? Maybe the function should account for that?

For example, since imageToImg() already takes a format and the image data, maybe this function should also just take "format" and "image" arguments. If we do that, we're able to calculate the exact size of the resulting HTML message -- not just an estimate.